shell/browser/electron_browser_client.cc

```diff
@@ -1054,7 +1061,7 @@ bool ElectronBrowserClient::HandleExternalProtocol(
     mojo::PendingRemote<network::mojom::URLLoaderFactory>* out_factory) {
   content::GetUIThreadTaskRunner({})->PostTask(
       FROM_HERE,
-      base::BindOnce(&HandleExternalProtocolInUI, url,
+      base::BindOnce(&HandleExternalProtocolInUI, url, frame_tree_node_id,
```

Comment:
Why not use `initiator_document` instead ? https://chromium-review.googlesource.com/c/chromium/src/+/3011560 added this argument, and it seems to keep track of the validity of the `RenderFrameHost` if navigation happens in the document.

Reply:
Changed